src/ape_plugins/_cli.py

```diff
@@ -168,7 +169,62 @@ def install(cli_ctx, skip_confirmation):
             # NOTE: Be *extremely careful* with this command, as it modifies the user's
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
-            subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"])
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"]
+            )
+            if result != 0:
+                cli_ctx.logger.error(f"Failed to add '{package_name}'.")
+                any_install_failed = True
+    if any_install_failed:
+        sys.exit(1)
+
+
+@cli.command(short_help="Uninstall all plugins in the local config file")
+@ape_cli_context()
+@skip_confirmation_option("Don't ask for confirmation to install the plugins")
+def uninstall(cli_ctx, skip_confirmation):
+    any_uninstall_failed = False
+    plugins = config.get_config("plugins") or []
+    for plugin in plugins:
+        # name schema is wrong
+        if plugin.startswith("ape"):
+            cli_ctx.abort(
+                f"Namespace 'ape' in '{plugin}' is not required. Please remove the pre-fix ape."
+            )
+            any_uninstall_failed = True
+
+        module_name, package_name = extract_module_and_package_install_names(plugin)
+        # check for installed check the config.yaml
+        if not is_plugin_installed(module_name):
+            cli_ctx.logger.warning(f"Plugin '{plugin}' is not installed.")
+            any_uninstall_failed = True
+
+        # if plugin is installed but not a 2nd class. It must be a third party
+        if module_name not in github_client.available_plugins and is_plugin_installed(plugin):
```

Comment:
I think we should make this condition happen first? That way, we don't both checking if the plugin is installed or not if is not an available plugin. And then, make this `if` an `elif`.